### PR TITLE
migration(crons): Backfill next check-in latest

### LINKED ISSUE #53958: Backfill the `next_checkin_latest` with the existing values from `next_checkin`.

### PR DESCRIPTION
**Pull request by rjo100**
Backfills `next_checkin_latest` with `next_checkin` if null

Closes https://github.com/getsentry/sentry/issues/53958

### CODE REVIEW

File: src/sentry/migrations/0527_backfill_next_checkin_latest.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+def backfill_next_checkin_latest(apps, schema_editor):
+    MonitorEnvironment = apps.get_model("sentry", "MonitorEnvironment")
+
+    for monitor_environment in RangeQuerySetWrapperWithProgressBar(
+        MonitorEnvironment.objects.all()
+    ):
+        if monitor_environment.next_checkin_latest is not None:
+            continue
+
+        monitor_environment.next_checkin_latest = monitor_environment.next_checkin
```
Comment (wedamija) [inline src/sentry/migrations/0527_backfill_next_checkin_latest.py:18]: Looks like `next_checkin` can be null, not sure if it is in practice?

**Review by wedamija — APPROVED** [review]
Only 17k rows here, should finish fast

File: src/sentry/migrations/0527_backfill_next_checkin_latest.py (hunk 1 above)
Comment (rjo100) [reply src/sentry/migrations/0527_backfill_next_checkin_latest.py:18]: it is for newly created monitor environments. will double check redash. but the main thing is pulling any old values into the new field if they haven't been updated. if both are null that's fine.

Comment (rjo100) [reply src/sentry/migrations/0527_backfill_next_checkin_latest.py:18]: yeah there are some where it's null

Comment (rjo100) [reply src/sentry/migrations/0527_backfill_next_checkin_latest.py:22]: updated logic

**Review by wedamija — APPROVED** [review]

**Pull request merged**